userdocs/src/community.md

```diff
@@ -7,20 +12,68 @@
+it into scope for 2021
+
+Current interaction with `eksctl` is imperative, we hope to add support for declarative
+configuration and cluster reconciliation via a new `eksctl apply -f config.yaml`
+command.  This model will additionally allow users to manage a cluster via a git repo.
 
-One or more release candidate(s) (RC) builds will be made available prior to each minor release. RC builds are intended only for testing purposes.
+A [WIP proposal](https://github.com/weaveworks/eksctl/blob/main/docs/proposal-007-apply.md)
+is already under consideration, to participate in the development of this feature
+please refer to the [tracking issue](https://github.com/weaveworks/eksctl/issues/2774)
+and our [proposal contributing guide](https://github.com/weaveworks/eksctl/blob/main/CONTRIBUTING.md#proposals).
 
-## Project Roadmap Themes
+### Flux v2 integration (GitOps Toolkit)
 
-### Cluster Quick Start with gitops
+In 2019 `eksctl` gave users a way to easily create a Gitops-ready ([Flux v1](https://docs.fluxcd.io/en/1.21.1/))
+cluster and to declare a set of pre-installed applications ([Quickstart profiles](https://eksctl.io/gitops-quickstart/#enabling-a-quick-start-profile))
+which can be managed via a git repo.
 
-It should be easy to create a cluster with various applications pre-installed, e.g. Weave Flux, Helm 2 (Tiller), AWS Load Balancer Controller. It should also be easy to manage these applications in a declarative way using config files in a git repo (with [gitops](https://www.weave.works/blog/what-is-gitops-really)).
+Since then, the practice of GitOps has matured, therefore `eksctl`'s support of
+GitOps will change to keep up with current standards.
 
-### Declarative configuration management for clusters
+The first stage of this is implementing support for [Flux v2](https://fluxcd.io/).
+This support already exists in an experimental form in `eksctl` (`eksctl enable flux`)
+and work there will carry on throughout the year.
+The next stage is to deprecate support for Flux v1, and expand the capacity of
+quickstart profiles. More information can be found [here](https://github.com/weaveworks/eksctl/issues/2963).
 
-One should be able to make EKS cluster configuration through declarative config files (`eksctl apply`). Additionally, they should be able to manage a cluster via a git repo.
+### Terraform
 
-### Cluster addons
+This is another in-demand feature which has finally made it to the list.
+Progress can be followed [here](https://github.com/weaveworks/eksctl/issues/1094).
 
-Understanding how the [add-ons spec by SIG Cluster Lifecycle](https://github.com/kubernetes/enhancements/pull/746) will evolve and how we can implement management of some cluster applications via the addon spec.
+An `eksctl` Terraform provider will let users manage their AWS clusters in the
+same format as for their other providers, without needing to understand the intricacies
+of EKS directly.
```

Comment:
Could be less definite? I.e. at the moment a provider is still just one of the options, right?

```suggestion
One option would be support for an `eksctl` Terraform provider that would let users manage their AWS clusters in the
same format as for their other providers, without needing to understand the intricacies of EKS directly.
```

Reply:
If we are still deciding on what we want to do wrt terraform, then i feel we probably should not mention it here but just have it in the researching section of the roadmap

Reply:
Ok, then my stance is don't mention it on this page